docs/_documentations/workingwithtemplates.md

```diff
@@ -92,11 +92,26 @@ Add your template sources to Codewind with the **Template Source Manager**.
    - **VS Code:** In the **Template Source Manager**, click **Add New**.
    - **Eclipse and IntelliJ:**
      - Right-click the connection in the Codewind Explorer view in Eclipse or the Codewind view in IntelliJ. Then, select **Manage Template Sources...**.
-     - After the **Manage Template Sources** wizard appears, click **Add...**.
+     - After the **Manage Template Sources** window appears, click **Add...**.
 2. Enter the URL and other information.
-   - **VS Code:** Enter the URL to your template source `index` file and click **Enter** to confirm.
+   - **VS Code:**
+     - Enter the URL to your template source `index` file and click **Enter** to confirm.
    - **Eclipse and IntelliJ:**
-     - In the **Add a Template Source** window, provide authentication information if necessary.
-     - To test the authentication information with the secured template repository, click **Test Template Source**.
-     - When you receive the message for the successful test, click **OK**.
-     - Then, click **Next** to reach the **Details** page. Complete the fields and click **Finish**.
\ No newline at end of file
+     - In the **Add a Template Source** window, enter the template source URL.
+3. Provide authentication if necessary.
+   - **VS Code:**
+     - Click **Authenticate**.
+     - Select whether to authenticate with a username and password or with an access token and enter the information.
+        - For GitHub, tokens that are used in Codewind must have the repo permission scope.
+     - Codewind tests the new source to make sure it can access it. After a successful test, enter a name and description for the new source if the source does not already have them. The new source appears in the Manage Sources page.
+   - **Eclipse and IntelliJ:**
+      - Enter the URL.
+      - If the template source requires authentication, select **Authentication required for this URL**, choose the authentication method, and complete the authentication details.
+        - **Logon authentication** requires a username and password.
+        - **Access token authentication** requires an access token.
+      - Click **Test Template Source** to check that the URL and, if included, the credentials are working.
+      - After you view the test results, click **OK**.
+      - Click **Next**.
+      - Enter a **Name** and **Description** for the template source.
```

Comment:
Similar to VSCode it should say `Enter a name and description for the new source if the source does not already have them.`